api/routes.go

```diff
@@ -0,0 +1,14 @@
+package api
+
+// buildHTTPRoutes registers all HTTP routes and their handlers.
+func (api *API) buildHTTPRoutes() {
+	// TODO Rate-limit these from the start to prevent brute force and other forms of abuse.
+	api.Router.POST("/login", api.userLoginHandler)
+	api.Router.POST("/user", api.userHandlerPOST)
+	api.Router.GET("/user/", api.userHandlerGET)
```

Comment:
```suggestion
	api.Router.GET("/user", api.userHandlerGET)
```
I think

Reply:
It doesn't really matter because the httprouter will redirect accordingly. But I agree it looks better. ;)